### ISSUE TITLE
Accordion lacks semantic markup and ARIA attributes

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#194]
## Problem

The accordion component is not accessible with a screenreader. The state of a section (open or closed) is not communicated, the aria-expanded attribute is missing.

## Suggestion

Follow the ARIA design pattern voor accordion.

## Articles en demo

* [Aditus: Accessible Accordion](https://www.aditus.io/patterns/accordion/)
* [ARIA Design Pattern for Accordion (Sections With Show/Hide Functionality)](https://www.w3.org/TR/wai-aria-practices-1.2/#accordion)
